client/my-sites/plans/jetpack-plans/product-store/items-list/products-list.tsx

```diff
@@ -17,7 +18,12 @@ export const ProductsList: React.FC< ProductsListProps > = ( {
 	const translate = useTranslate();
 
 	const allItems = useMemo(
-		() => getSortedDisplayableProducts( [ ...popularProducts, ...otherProducts ] ),
+		() =>
+			getSortedDisplayableProducts( [
+				...popularProducts,
+				...otherProducts,
+				slugToSelectorProduct( 'jetpack_creator_yearly' )!,
```

Comment:
Since Jetpack Creator is technically a "Plan"/Bundle and not a Product, are we intentionally wanting to show it in the list of Products and not as a Bundle?, up there with Security & Complete?
If so, that's fine, but unfortunately adding it in this way will not allow it to have the ability to show as an "owned" and manageable/upgradeable (when logged in). For example, If I purchased/owned Jetpack Creator (monthly), we would expect it to show on the plans/pricing page like this:
![Markup 2023-11-01 at 03 39 44](https://github.com/Automattic/wp-calypso/assets/11078128/399fb7ce-d237-4056-b539-cc78edd8ecab)

The way it is added in here does not allow it to have any owned/manageable state.
To have the Creator bundle show correctly in the product list with owned state, We'll want to remove it from here and instead include it in `use-get-plans-grid-products.ts`. It's kinda tricky because we're treating a plan/bundle as if it were a "product", so therefore we'll need to include the JETPACK_CREATOR_PLANS when calculating the site's `purchasedProducts`:

use-get-plans-grid-products.ts - line 45
```
// Owned products from direct purchases
const purchasedProducts =
	useSelector( ( state ) => getSiteProducts( state, siteId ) )
		?.map( ( { productSlug } ) => productSlug )
		// If the Creator plan/bundle has been purchased we want to show it in the list of products.
		.concat( currentPlan ? [ currentPlan ] : [] )
		.filter( ( productSlug ) =>
			(
				[ ...JETPACK_PRODUCTS_LIST, ...JETPACK_CREATOR_PLANS ] as ReadonlyArray< string >
			 ).includes( productSlug )
		) ?? [];
```

And then if it's not owned/purchased, we include it in the array of `avaliableProducts`, just like we do with all the other products:
```
// If the user does not own the Creator plan, include it in available products
if (
	! ownedProducts.some( ( ownedProduct ) =>
		( JETPACK_CREATOR_PLANS as ReadonlyArray< string > ).includes( ownedProduct )
	)
) {
	availableProducts = [ ...availableProducts, ...JETPACK_CREATOR_PLANS ];
}
```

Reply:
Thanks for coming up with solution @elliottprogrammer 🙏 Indeed, we want to display it as a product card for now. I know it creates technical complications, but we don't want to expose it as a bundle either.

I've implemented your solution to the code.